taipy/core/_orchestrator/_abstract_orchestrator.py

```diff
@@ -21,6 +21,16 @@
 class _AbstractOrchestrator:
     """Creates, enqueues, and orchestrates jobs as instances of `Job^` class."""
 
+    @property
+    @abstractmethod
+    def jobs_to_run(self):
+        pass
+
+    @property
+    @abstractmethod
+    def blocked_jobs(self):
+        pass
```

Comment:
I don't think these 2 functions were implemented right?

Reply:
Right. These are two properties. 
The dispatchers are using these two properties. However, since dispatchers take an abstract orchestrator as input, the two properties should be declared at the abstract orchestrator level and not only at the orchestrator implementation level.

Reply:
ahhh my bad, I looked too quickly and thought this is the dispatcher file, was wondering why we're declaring these 2 properties in the dispatchers haha

hmm in that case, I think we should still define it in the orchestrator no? or is it not needed because they already exist as class attribute?

Reply:
They are already defined as attributes, indeed.